hub/apps/design/input/drag-and-drop.md

```diff
@@ -51,7 +51,8 @@ For other content, you'll need to handle the [**DragStarting**](/uwp/api/windows
 
 ## Enable dropping
 
-The following markup shows how to set a specific area of the app as valid for dropping by using the [**AllowDrop**](/uwp/api/windows.ui.xaml.uielement.allowdrop) in XAML. If a user tries to drop somewhere else, the system won't let them. If you want users to be able to drop items anywhere on your app, set the entire background as a drop target.
+The following markup shows how to set a specific area of the app as valid for dropping by using the [**AllowDrop**](/uwp/api/windows.ui.xaml.uielement.allowdrop) in XAML. If a user tries to drop somewhere else, the system won't let them. 
+If you want users to be able to drop items anywhere on your app, set the entire background as a drop target. Note that the background property defaults to null, which will not allow drops to happen (as there is nothing to hit-test), so make sure you set the background to a brush (a transparent brush will work too).
```

Comment:
The background _may_ be null, depending on whether the app has already set it for other reasons or whether the selected element is one which gets a background by default.

I suggest something like this:
```
Drops can only happen in areas where the `AllowDrop="True"` element receives pointer input. If the element doesn't have a background, you may need to set `Background="Transparent"` to ensure it receives input over its entire bounds.
```
If there is a doc about null/transparent to point to, even better. I didn't readily find anything.

Reply:
thanks, good precision to add :)

Reply:
```suggestion
If you want users to be able to drop items anywhere on your app, set the entire background as a drop target. Drops can only happen in areas where the `AllowDrop="True"` element receives pointer input. If the element doesn't have a background, you may need to set `Background="Transparent"` to ensure it receives input over its entire bounds.
```